App.js

```diff
@@ -1,26 +1,28 @@
 import React from 'react'
+import data from './data.json'
+
+import { Alfabet } from './components/Alfabet.js'
 import styled from 'styled-components/native'
```

Comment:
`styled-components` is an outer dependency, therefore it should be placed next to `react`